src/main/kotlin/domain/LottoGenerator.kt

```diff
@@ -0,0 +1,36 @@
+package domain
+
+import domain.model.PurchaseMoney
+import domain.model.lotto.Lotto
+
+class LottoGenerator(
+    private val numberGenerator: () -> Set<Int> = {
+        (MINIMUM_NUMBER..MAXIMUM_NUMBER).drawLotteryNumbers()
+    }
+) {
+
+    fun generateLottos(purchaseMoney: PurchaseMoney): List<Lotto> {
+        purchaseMoney.validateMoneyUnit()
+
+        return List(purchaseMoney.getNumberOfLottos()) {
+            Lotto(numberGenerator())
+        }
+    }
+
+    private fun PurchaseMoney.validateMoneyUnit() = require(this.money % domain.LottoGenerator.LOTTO_PRICE == domain.LottoGenerator.ZERO) {
+        domain.LottoGenerator.NUMBER_UNIT_ERROR
+    }
```

Comment:
```suggestion
    private fun PurchaseMoney.validateMoneyUnit() = require(this.money % LOTTO_PRICE == ZERO) {
        NUMBER_UNIT_ERROR
    }
```
불필요한 패키지를 제거해보면 어떨까요? (이하 관련 내용 동일)